libcontainer/cgroups/utils.go

```diff
@@ -479,6 +479,26 @@ func RemovePaths(paths map[string]string) (err error) {
 	return fmt.Errorf("Failed to remove paths: %v", paths)
 }
 
+// RemovePathUnified aims to remove cgroup path recursively
+// Because there may be subcgroups in it.
+func RemovePathUnified(path string) error {
+	infos, err := ioutil.ReadDir(path)
+	if err != nil {
+		return err
+	}
+	for _, info := range infos {
+		if info.IsDir() {
+			if err = RemovePathUnified(filepath.Join(path, info.Name())); err != nil {
```

Comment:
Maybe here should ignore NotExist error?

Reply:
Yes